src/components/canrawview/uniquefiltermodel.h

```diff
@@ -3,21 +3,50 @@
 
 #include <QSortFilterProxyModel>
 
+/**
+*   @brief This class provides a filter model between source model and table view, which passes through only frames with
+*          newest unique ID and direction values
+*/
 class UniqueFilterModel : public QSortFilterProxyModel {
     Q_OBJECT
 public:
     explicit UniqueFilterModel(QObject* parent = 0);
 
+    /**
+    *   @brief  Updates filter with unique values, on reception of each frame
+    *   @param  current frame ID
```

Comment:
First item after param must be argument name. E.g:

    /**
    *   @brief  Configures CAN BUS backend and interface
    *
    *   This function is used to configure QtCanBus class.
    *
    *   @param  backend one of backends supported by QtCanBus class
    *   @param  iface CAN BUS interface index (e.g. can0 for socketcan backend)
    *   @return true on success, false of failure
    */
    bool init(const QString& backend, const QString& iface);